tasks/buildah.yaml

```diff
@@ -113,12 +113,30 @@ spec:
     workingDir: $(workspaces.source.path)
     securityContext:
       runAsUser: 0
+  - image: registry.access.redhat.com/ubi8/python-39:1-51
+    name: convert-cyclonedx-to-purl
+    script: |
+      #!/bin/python3
+      import sys
+      import json
+
+      with open("./sbom.json") as f:
+        data = json.load(f)
+
+      purls = [component["purl"] for component in data["components"] if "purl" in component]
+
+      with open("sbom-purl", "w") as output_file:
+        for purl in purls:
+          output_file.write(f"{purl}\n")
+    workingDir: $(workspaces.source.path)
+    securityContext:
+      runAsUser: 0
   - image: $(params.BUILDER_IMAGE)
     name: inject-sbom
     resources: {}
     script: |
       container=$(buildah from --storage-driver=$(params.STORAGE_DRIVER) --pull-never $(params.IMAGE))
-      buildah --storage-driver=$(params.STORAGE_DRIVER) copy $container sbom.json /root/
+      buildah --storage-driver=$(params.STORAGE_DRIVER) copy $container sbom.json sbom-purl /root/
```

Comment:
I think it would be good to change it to `sbom-purl.json`, since it's still a json file.

Reply:
And maybe change the other to `sbom-cyclonedx.json`?

Reply:
changed both names